.release.cloudbuild.yaml

```diff
@@ -24,6 +24,38 @@ steps:
       # Parse major minor version and save to a file for reusing in other steps.
       echo $TAG_NAME | sed -e "s#[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)#\1.\2#" > /workspace/mm.ver
 
+# Pull and retag images for pipeline components
```

Comment:
Are you sure we need to do this every time? The code of these components rarely changes. Some people are getting a bit confused since we release new images and update the components and samples, while there are no real changes.

Reply:
We've been doing these previously. So I'm just re-implementing current behavior.

I don't feel we should always release them every time, but releasing them each time is easier to manage (and there are minor changes e.g. the base image can receive updates to gcloud.)

If we decide they shouldn't be released every time, we should probably put components on a different release schedule, or just adhoc release them.

Reply:
>If we decide they shouldn't be released every time, we should probably put components on a different release schedule, or just adhoc release them.

It might be fine to release them ad-hoc like we do now.

Reply:
hold on... I think we need some clarification.
I've been following release playbook and it doesn't say component/sample release part is optional, so I (and others I think) always release them during periodic release.

but you mentioned we release them ad-hoc now.

Should the release playbook be updated?

Reply:
This part is optional.
Changes to the code used in those containers are now pretty rare and re-releasing components without any changes confuses users (I've received some feedback).

I thought this was reflected in the playbook, but if it isn't then let's update the playbook.

Reply:
SGTM, please update the playbook. I will adjust here accordingly